### PR TITLE
awoa 26 scenes, media and components

### PR DESCRIPTION
**Pull request by tushararora**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -332,6 +343,18 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [VR Space Builder](http://vr.greenbushlabs.com/) - Simple VR builder for kids using a form to import models and images.
 - [WebVR API Simulation Extension](https://chrome.google.com/webstore/detail/webvr-api-emulation/gbdnpaebafagioggnhkacnaaahpiefil) - Run WebVR 1.0 content without a supported headset or even a compatible browser/platform. Great for development.
 - [WebVR Solar System](http://vrspace.jmvisualcreativity.es/) - Solar System using A-Frame.
+- [Track](https://samsunginternet.github.io/a-frame-demos/racer.html) - Zooming on the track with the ocean water reflecting the sunset sky. [[Code](https://github.com/SamsungInternet/a-frame-demos/blob/gh-pages/racer.md)]
```
Comment (cvan) [inline README.md]: can you fix this to point to https://samsunginternet.github.io/a-frame-demos/racer/ and https://github.com/SamsungInternet/a-frame-demos/blob/gh-pages/racer/index.html, respectively?

File: README.md [hunk 2]
```diff
@@ -332,6 +343,18 @@ Use React with A-Frame. Share your A-Frame React Components!
...
+- [Interactive 360 Videos](http://s3.amazonaws.com/hapyak_demos/interactive360video/edit.html) - Upload and edit 360 interactive videos by overlaying DOM elements.
+- [Unity Exporter to A-Frame](https://github.com/if1live/unity-scene-web-exporter) - Export Unity scenes to three.js or A-Frame with lightmapping.
+- [Wordpress Template](http://www.construktiv.de/imagine/webvr-mit-a-frame/) - Wordpress template that displays the four most recent posts in VR.
+- [Web Speech API Test](https://codepen.io/bryik/pen/mErOOR?editors=0010) - A demo of the Speech Recognition API to change object colors.
+- [Apollo 11 Scan](http://math.nist.gov/~SRessler/aframe/ebd/projects/apollo/) - View the inside of the Apollo 11 Command Module thanks for the National Air and Space Museum.
+- [Olympic Rings](https://trys.github.io/aframe/2/) - A-Frame gets the gold at Rio 2016.
+- [Covo](https://s3-us-west-1.amazonaws.com/geopogo/covo/aframe_0_1_0/index.html) - Architectural visualization with baked global illumination.
+- [Humanity](http://codepen.io/davatron5000/pen/RRmVQz) - Day 8 of Dave’s 30 Days of WebVR.
```
Comment (cvan) [inline README.md]: could you use the `https://` URL?

File: README.md [hunk 3]
```diff
@@ -332,6 +343,18 @@ Use React with A-Frame. Share your A-Frame React Components!
...
+- [Heart](https://s3.amazonaws.com/vr-asset-repo/heart_demo_slack.html) - An educating and interactive tour of the heart. But a life is at stake.
+- [Miradas 360](https://miradas360.github.io/miradas360-01/) - 360 experiences created by media production teams. [[Code](https://github.com/Miradas360/miradas360-01)]
+- [Interactive 360 Videos](http://s3.amazonaws.com/hapyak_demos/interactive360video/edit.html) - Upload and edit 360 interactive videos by overlaying DOM elements.
+- [Unity Exporter to A-Frame](https://github.com/if1live/unity-scene-web-exporter) - Export Unity scenes to three.js or A-Frame with lightmapping.
+- [Wordpress Template](http://www.construktiv.de/imagine/webvr-mit-a-frame/) - Wordpress template that displays the four most recent posts in VR.
+- [Web Speech API Test](https://codepen.io/bryik/pen/mErOOR?editors=0010) - A demo of the Speech Recognition API to change object colors.
+- [Apollo 11 Scan](http://math.nist.gov/~SRessler/aframe/ebd/projects/apollo/) - View the inside of the Apollo 11 Command Module thanks for the National Air and Space Museum.
```
Comment (cvan) [inline README.md]: `thanks for` -> `thanks to`?

File: README.md [hunk 4]
```diff
@@ -145,6 +154,8 @@ Third-party components (of the entity-component-system pattern). To create and s
 - [Trigger Box](https://github.com/Utopiah/aframe-triggerbox-component) - A component that emits an event when it enters or leaves a predefined area.
 - [L-System](https://github.com/nylki/aframe-lsystem-component) - A component for rendering [Lindenmayer](https://en.wikipedia.org/wiki/L-system) systems.
 - [Terrain Model Component](https://github.com/bryik/aframe-terrain-model-component) - Generating terrains using TerrainLoader
+- [Orbit Controls Component](https://github.com/tizzle/aframe-orbit-controls-component) - Allow desktop users to rotate the camera around an object.
+- [Sprite Component](https://github.com/tizzle/aframe-sprite-component) - Creates bitmap images that always faces the camera.
```
Comment (cvan) [inline README.md]: `faces` -> `face`

File: README.md [hunk 5]
```diff
@@ -50,6 +50,11 @@ Learn about A-Frame.
 - [Lightmapping on the Server](https://medium.com/@bnolan/lightmapping-on-the-server-5485177ef2fa) - Server-side lightmapping with Blender
 - [Scene Export](https://medium.com/@bnolan/scene-export-89cc1c985ac8) - Exporting A-Frame-based scenes in SceneVR
 - [Using Thingiverse Files in A-Frame](https://medium.com/@bnolan/using-thingiverse-files-in-a-frame-50641ca3f721)
+- [Building 3D Games on the Web](https://hacks.mozilla.org/2016/06/basics-of-building-3d-games-on-the-web/) - Basics of building 3D games on the web
+- [Getting Started With A-Frame](http://www.tomango.co.uk/thinks/getting-started-aframe/) - Building the Olympic rings using A-Frame
+- [Head Tracked Transformations](https://medium.com/humane-virtuality/head-tracked-transformations-e7102d3c9789)- How to look behind an object in VR if you can’t walk around it
```
Comment (cvan) [inline README.md]: space after `-`

**Comment by cvan** [comment]
thanks for opening this PR 👍
just left a few nits - let me know when you've updated

**Comment by cvan** [comment]
also, might want to check out https://github.com/aframevr/aframe-site/pull/337/files - I fixed some typos. I think you're good though.

@ngokevin: if you'd like, as I know you've said before, feel free to merge this, and we can fix later. up to you. thanks!

**Comment by tushararora** [comment]
@cvan Thanks for pointing out typos. Here is the commit https://github.com/aframevr/awesome-aframe/pull/98/commits/5f052422991d4ef60fb5912e4c09abc5aba44a8c

**Pull request merged**